src/Services/Concerns/MocksRequests.php

```diff
@@ -68,6 +76,16 @@ public function fake($callback = null)
      */
     public function saveSnapshots()
     {
+
+        // clear snapshots if it is not cleared and RECREATE_SNAPSHOTS=true
+        if (! self::$snapshotsCleared && env('RECREATE_SNAPSHOTS', false)) {
+            if (file_exists($this->recordingsFilepath)) {
+                file_put_contents($this->recordingsFilepath, '');
+                // mark snapshots cleared
+                self::$snapshotsCleared = true;
+            }
+        }
+
```

Comment:
open for recommendations, this function is being called from every service, so need to prevent snapshot cleaning on every call but first.

Reply:
Sure, I almost forgot about bootstrapping, will be checking that

Reply:
make sense I also wanted a place to run it once, will try

Reply:
check the bootstrapping file I've included now